### PR TITLE
Add testing with PHP 8.0

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by mvorisek** [comment]
@DarkSide666 wdyt? `_param` still accepts any scalar type (I think it is better so than casting it somewhere before)

tested/passing with `data` and `ui`

File: src/Expression.php [hunk 1]
```diff
@@ -376,17 +358,12 @@ protected function _escape($value)
      * query rendering. Consider using `_consume()` instead, which will
      * also handle nested expressions properly.
      *
-     * @param string|array $value String literal or array of strings containing input data
+     * @param string|int|float $value
```
Comment (DarkSide666) [inline src/Expression.php:361]: I guess it should be `mixed` type here.

**Review by DarkSide666 — APPROVED** [review]
Looks fine.

File: src/Expression.php (hunk 1 above)
Comment (mvorisek) [reply src/Expression.php:361]: mixed means any, I think it should be string or scalar (we should cast everything to string in dsql anyway...)

Comment (mvorisek) [reply src/Expression.php:361]: image

**Pull request merged**